lib/functions-custom.php

```diff
@@ -126,3 +126,35 @@ function render_front_splash($front_id) {
 <?php
   }
 }
+
+// Render Support page form
+function render_support_form($form_options, $form_recipient) {
+?>
+<form id="support-form" class="grid-row align-items-end" method="post" action="<?php echo get_stylesheet_directory_uri() . '/lib/form-mailer.php'; ?>">
+  <div id="i-want-to" class="grid-item">
+    I want to
+  </div>
+  <div class="grid-item grid-column flex-grow">
+    <input id="support-form-email" class="form-element margin-top-basic" type="email" placeholder="my email" name="email"/>
```

Comment:
Form elements should have `required` value for browser validation. That would be more than enough validation considering serverside sanitization imo